rules/windows/privilege_escalation_reg_service_imagepath_mod.toml

```diff
@@ -0,0 +1,202 @@
+    "SYSTEM\\*ControlSet*\\Services\\IKEEXT",
+    "SYSTEM\\*ControlSet*\\Services\\InstallService",
+    "SYSTEM\\*ControlSet*\\Services\\iphlpsvc",
+    "SYSTEM\\*ControlSet*\\Services\\IsmServ",
+    "SYSTEM\\*ControlSet*\\Services\\LanmanServer",
+    "SYSTEM\\*ControlSet*\\Services\\MSiSCSI",
+    "SYSTEM\\*ControlSet*\\Services\\NcbService",
+    "SYSTEM\\*ControlSet*\\Services\\Netlogon",
+    "SYSTEM\\*ControlSet*\\Services\\Netman",
+    "SYSTEM\\*ControlSet*\\Services\\NtFrs",
+    "SYSTEM\\*ControlSet*\\Services\\PlugPlay",
+    "SYSTEM\\*ControlSet*\\Services\\Power",
+    "SYSTEM\\*ControlSet*\\Services\\PrintNotify",
+    "SYSTEM\\*ControlSet*\\Services\\ProfSvc",
+    "SYSTEM\\*ControlSet*\\Services\\PushToInstall",
+    "SYSTEM\\*ControlSet*\\Services\\RSoPProv",
+    "SYSTEM\\*ControlSet*\\Services\\sacsvr",
+    "SYSTEM\\*ControlSet*\\Services\\SENS",
+    "SYSTEM\\*ControlSet*\\Services\\SensorDataService",
+    "SYSTEM\\*ControlSet*\\Services\\SgrmBroker",
+    "SYSTEM\\*ControlSet*\\Services\\ShellHWDetection",
+    "SYSTEM\\*ControlSet*\\Services\\shpamsvc",
+    "SYSTEM\\*ControlSet*\\Services\\StorSvc",
+    "SYSTEM\\*ControlSet*\\Services\\svsvc",
+    "SYSTEM\\*ControlSet*\\Services\\swprv",
+    "SYSTEM\\*ControlSet*\\Services\\SysMain",
+    "SYSTEM\\*ControlSet*\\Services\\Themes",
+    "SYSTEM\\*ControlSet*\\Services\\TieringEngineService",
+    "SYSTEM\\*ControlSet*\\Services\\TokenBroker",
+    "SYSTEM\\*ControlSet*\\Services\\TrkWks",
+    "SYSTEM\\*ControlSet*\\Services\\UALSVC",
+    "SYSTEM\\*ControlSet*\\Services\\UserManager",
+    "SYSTEM\\*ControlSet*\\Services\\vm3dservice",
+    "SYSTEM\\*ControlSet*\\Services\\vmicguestinterface",
+    "SYSTEM\\*ControlSet*\\Services\\vmicheartbeat",
+    "SYSTEM\\*ControlSet*\\Services\\vmickvpexchange",
+    "SYSTEM\\*ControlSet*\\Services\\vmicrdv",
+    "SYSTEM\\*ControlSet*\\Services\\vmicshutdown",
+    "SYSTEM\\*ControlSet*\\Services\\vmicvmsession",
+    "SYSTEM\\*ControlSet*\\Services\\vmicvss",
+    "SYSTEM\\*ControlSet*\\Services\\vmvss",
+    "SYSTEM\\*ControlSet*\\Services\\VSS",
+    "SYSTEM\\*ControlSet*\\Services\\w3logsvc",
+    "SYSTEM\\*ControlSet*\\Services\\W3SVC",
+    "SYSTEM\\*ControlSet*\\Services\\WalletService",
+    "SYSTEM\\*ControlSet*\\Services\\WAS",
+    "SYSTEM\\*ControlSet*\\Services\\wercplsupport",
+    "SYSTEM\\*ControlSet*\\Services\\WerSvc",
+    "SYSTEM\\*ControlSet*\\Services\\Winmgmt",
+    "SYSTEM\\*ControlSet*\\Services\\wisvc",
+    "SYSTEM\\*ControlSet*\\Services\\wmiApSrv",
+    "SYSTEM\\*ControlSet*\\Services\\WPDBusEnum",
+    "SYSTEM\\*ControlSet*\\Services\\WSearch"
+  ) and
+  not (
+    registry.data.strings : (
+        "C:\\system32\\*.exe",
+        "%systemroot%\\system32\\*.exe",
+        "%windir%\\system32\\*.exe",
+        "%SystemRoot%\\system32\\svchost.exe -k *",
+        "%windir%\\system32\\svchost.exe -k *"
+    ) and
+        not registry.data.strings : (
```

Comment:
better use `not endswith~(registry.data.strings, "cmd.exe", "cscript.exe" ...)` for performance.

Reply:
https://github.com/elastic/detection-rules/pull/3757#discussion_r1631092363